.prettierrc

```diff
@@ -1,5 +1,4 @@
 {
-  "printWidth": 120,
```

Comment:
Why is the printWidth removed?

Reply:
Switched this setting to [default value](https://prettier.io/docs/en/options.html#print-width)
to mach the same code style as in [Platform project](https://github.com/visual-knight/platform-community-edition/blob/master/.prettierrc)

Reply:
We should commit to a value and use this here and in the platform repository. Problem without one is that it would use the one set by the IDE and this could be different.

Which one you would prefer?
1: 120
2: 100


Smaller is getting to short in my opinion

Reply:
120 is fine for me, value is reverted